Index: ivozprovider/kamailio/trunks/config/kamailio.cfg
===================================================================
--- ivozprovider.orig/kamailio/trunks/config/kamailio.cfg
+++ ivozprovider/kamailio/trunks/config/kamailio.cfg
@@ -102,8 +102,8 @@ log_facility=LOG_LOCAL0
 onsend_route_reply=yes
 
 # Add custom Server header
-server_header="Server: PowerPBX v2.9"
-user_agent_header="User-Agent: PowerPBX v2.9"
+server_header="Server: Irontec IvozProvider v2.9"
+user_agent_header="User-Agent: Irontec IvozProvider v2.9"
 
 ####### Modules Section ########
 
Index: ivozprovider/kamailio/users/config/kamailio.cfg
===================================================================
--- ivozprovider.orig/kamailio/users/config/kamailio.cfg
+++ ivozprovider/kamailio/users/config/kamailio.cfg
@@ -126,8 +126,8 @@ log_facility=LOG_LOCAL0
 onsend_route_reply=yes
 
 # Add custom Server header
-server_header="Server: PowerPBX v2.9"
-user_agent_header="User-Agent: PowerPBX v2.9"
+server_header="Server: Irontec IvozProvider v2.9"
+user_agent_header="User-Agent: Irontec IvozProvider v2.9"
 
 ####### Modules Section ########
 
Index: ivozprovider/scheme/app/DoctrineMigrations/Version20190126162606.php
===================================================================
--- /dev/null
+++ ivozprovider/scheme/app/DoctrineMigrations/Version20190126162606.php
@@ -0,0 +1,26 @@
+<?php
+ namespace Application\Migrations;
+ use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+ /**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20190126162606 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE ApplicationServers ADD grpid INT UNSIGNED DEFAULT 1');
+     }
+     /**
+     * @param Schema $schema
+     */
+    public function down(Schema $schema)
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE ApplicationServers DROP grpid');
+     }
+}
