Index: ivozprovider/profiles/as/root/.bashrc.d/400-application-server
===================================================================
--- ivozprovider.orig/profiles/as/root/.bashrc.d/400-application-server
+++ ivozprovider/profiles/as/root/.bashrc.d/400-application-server
@@ -1,5 +1,5 @@
 # Fancy Shell prompt
-export PS1='${debian_chroot:+($debian_chroot)}\[\033[01;31m\]\u@\H\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+export PS1='${debian_chroot:+($debian_chroot)}\[\033[01;31m\]\u@\H\[\033[00m\]:\[\033[01;35m\]\w\[\033[00m\]\$ '
 
 # Some useful aliases
 alias ast='asterisk -rvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv'
Index: ivozprovider/profiles/common/root/.bashrc.d/150-common-custom
===================================================================
--- /dev/null
+++ ivozprovider/profiles/common/root/.bashrc.d/150-common-custom
@@ -0,0 +1,2 @@
+# Add custom aliases for frequently used commands
+alias sc='systemctl'
+
Index: ivozprovider/profiles/data/root/.bashrc.d/300-data-system
===================================================================
--- ivozprovider.orig/profiles/data/root/.bashrc.d/300-data-system
+++ ivozprovider/profiles/data/root/.bashrc.d/300-data-system
@@ -1,2 +1,2 @@
 # Fancy Shell prompt
-export PS1='${debian_chroot:+($debian_chroot)}\[\033[01;34m\]\u@\H\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+export PS1='${debian_chroot:+($debian_chroot)}\[\033[01;35m\]\u@\H\[\033[00m\]:\[\033[01;35m\]\w\[\033[00m\]\$ '
Index: ivozprovider/profiles/portal/root/.bashrc.d/200-portal
===================================================================
--- ivozprovider.orig/profiles/portal/root/.bashrc.d/200-portal
+++ ivozprovider/profiles/portal/root/.bashrc.d/200-portal
@@ -1,3 +1,3 @@
 # Fancy Shell prompt
-export PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\H\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+export PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\H\[\033[00m\]:\[\033[01;35m\]\w\[\033[00m\]\$ '
 
Index: ivozprovider/profiles/proxy/root/.bashrc.d/500-proxy-system
===================================================================
--- ivozprovider.orig/profiles/proxy/root/.bashrc.d/500-proxy-system
+++ ivozprovider/profiles/proxy/root/.bashrc.d/500-proxy-system
@@ -1,5 +1,5 @@
 # Fancy Shell prompt
-export PS1='${debian_chroot:+($debian_chroot)}\[\033[01;33m\]\u@\H\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+export PS1='${debian_chroot:+($debian_chroot)}\[\033[01;33m\]\u@\H\[\033[00m\]:\[\033[01;35m\]\w\[\033[00m\]\$ '
 
 # Watch kamailio logs as in tail -f
 alias kamtail='journalctl -u kamailio@* -f -a | kamlog'
Index: ivozprovider/profiles/proxy/root/.bashrc.d/550-proxy-system-custom
===================================================================
--- /dev/null
+++ ivozprovider/profiles/proxy/root/.bashrc.d/550-proxy-system-custom
@@ -0,0 +1,20 @@
+# Custom aliases
+
+ # override previous alias set in common package
+unalias sc
+
+ ## EXAMPLES
+# "sc status ku" is alias for "systemctl --no-pager status kamailio@users.service -l"
+# "sc status kt" is alias for "systemctl --no-pager status kamailio@trunks.service -l"
+# "sc status firewalld" is alias for "systemctl status firewalld"
+
+ sc ()
+{
+	if [[ $2 == 'ku' ]]; then
+		systemctl --no-pager $1 kamailio@users.service -l
+	elif [[ $2 == 'kt' ]]; then
+		systemctl --no-pager $1 kamailio@trunks.service -l
+	else
+		systemctl --no-pager $1 $2 -l
+	fi
+}
+
